Dockerfile.ubuntu.18.04

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2019, Intel Corporation
+#
+# 'recipe' for Docker to build an Debian package
+#
+# Pull base image
+FROM ubuntu:18.04
+Maintainer John.E.Malmberg <John.E.Malmberg@intel.com>
+
+# Install basic tools
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+            autoconf bash curl debmake debhelper dpkg-dev doxygen gcc \
+            git locales make patch rpm
+
+# isa-l_crypto
+# isa-l_crypto needs a debhelper from backports
```

Comment:
This was true for isa-l.  Is it true for isa-l_crypto?

Reply:
To be honest, I don't know. isa-l_crypto is very similar to isa-l, so most of this PR is just copy/pasted. I'm not sure why it was true for isa-l, so I'm not exactly sure how to go about finding out if it's true for isa-l_crypto. Any suggestions?